protected/humhub/components/access/ControllerAccess.php

```diff
@@ -244,6 +244,12 @@ public function init()
             'reason' => Yii::t('error', 'Login required for this section.'),
             'code' => 401
         ]);
+        $this->registerValidator([
+            self::RULE_LOGGED_IN_ONLY => 'validateMaintenanceMode',
+            'reason' => Yii::t('error', 'Only admins have an access to the site on maintenance mode.'),
```

Comment:
Here as well `an` not really needed

Reply:
Commit https://github.com/humhub/humhub/pull/4773/commits/23a9ed3c0a5750e569d9480669fc7fe0d6f52ae1